rest/admin_api.go

```diff
@@ -148,10 +148,27 @@ func (h *handler) handleReplicate() error {
 	}
 
 	params, cancel, _, err := h.readReplicationParametersFromJSON(body)
+	if err != nil {
+		return err
+	}
 
+	response, err := h.server.HTTPClient.Get(params.GetTargetDbUrl())
```

Comment:
Would it make sense to do this in `validateReplicationParameters()`?

Reply:
This is ran during startup which would prevent SG starting if a db wasn't available. We decided against this and only to validate during REST.